meinberlin/apps/dashboard2/forms.py

```diff
@@ -251,3 +252,20 @@ class Meta:
         required_for_project_publish = ['polygon']
         # widgets = map_models.AreaSettings.widgets()
         widgets = {'polygon': MapChoosePolygonWithPresetWidget}
+
+
+class CategoryForm(forms.ModelForm):
+    name = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Category')}
+    ))
+
+    class Meta:
+        model = category_models.Category
+        exclude = ('module',)
```

Comment:
i don't like "exclude" definitions that much. i'd prefer to set `fields = ['name']`